### PR TITLE
Vulnerable Cancer Patients

### PR DESCRIPTION
**Pull request by CoderOMaster**
# Pull Request for ML-Crate 💡

## Issue Title:  Issue No 498

- **Info about the related issue (Aim of the project)** : 
- **Name:**  Keshav Arora
- **GitHub ID:**  CoderOMaster
- **Idenitfy yourself:**  JWOC 2024
**Please Mark this as hard:)**

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Not every issue will mark as Hard, if it goes like this every issue should be of Hard category then.

- Project folder name should be same as the issue name with no underscore.
- Put some EDA before going in to the models. Such as Predicted datapoints plot, intensity distribution plot etc.
- Make the README a bit attractive.

@CoderOMaster

**Comment by CoderOMaster** [comment]
I understand sir.
I will try my best to implement these

**Comment by CoderOMaster** [comment]
@abhisheks008 
Please check i have done the necessary changes now

File: Vulnerable Cancer Patient Analysis/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+## Models and Accuracies
+
+| Model             | Accuracy   | 
+| ----------------- |:----------:| 
+| Roberta           | 0.79       |                    
+| LSTM              | 0.72       |                    
+| Neural Network    | 0.67       |                    
+| Logistic Regression| 0.71      |                    
+
+**NOTE: No graphs were plotted**
```
Comment (abhisheks008) [inline Vulnerable Cancer Patient Analysis/README.md:39]: You have plotted graphs. Remove this and add Visualization heading under that put the graph of intensity distribution. After that Models Applied will come.

File: Vulnerable Cancer Patient Analysis/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+**ROBERTA MODEL IN TENSORFLOW AND PYTORCH WERE PARTIALLY TRAINED DUE TO RESOURCE LIMITATIONS; RESULTS FROM 3 EPOCHS WILL BE PROVIDED**
+
+**IMAGE**
```
Comment (abhisheks008) [inline Vulnerable Cancer Patient Analysis/README.md:43]: This is creating some styling problem, put `<br>` over here.

File: Vulnerable Cancer Patient Analysis/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+![Alt Text](./Images/confusion_matrix.png)
+
+![Alt Text](./Images/intensity.png)
+
```
Comment (abhisheks008) [inline Vulnerable Cancer Patient Analysis/README.md:47]: Missing Conclusion.

**Review by abhisheks008 — CHANGES_REQUESTED** [review]

File: Vulnerable Cancer Patient Analysis/README.md (hunk 1 above)
Comment (CoderOMaster) [reply Vulnerable Cancer Patient Analysis/README.md:39]: What exactly am I supposed to do? I don't get it please

File: Vulnerable Cancer Patient Analysis/README.md (hunk 3 above)
Comment (CoderOMaster) [reply Vulnerable Cancer Patient Analysis/README.md:47]: Conclusion as in? 
I have the result plotted

File: Vulnerable Cancer Patient Analysis/README.md (hunk 1 above)
Comment (abhisheks008) [reply Vulnerable Cancer Patient Analysis/README.md:39]: You have written no graphs were plotted that's why I told you to remove that line and make the changes accordingly.

File: Vulnerable Cancer Patient Analysis/README.md (hunk 3 above)
Comment (abhisheks008) [reply Vulnerable Cancer Patient Analysis/README.md:47]: After finishing a project don't you do some conclusion, this this things we can conclude or just left the project without putting any conclusion?

Comment (CoderOMaster) [reply Vulnerable Cancer Patient Analysis/README.md:46]: Done ✅

File: Vulnerable Cancer Patient Analysis/README.md (hunk 1 above)
Comment (CoderOMaster) [reply Vulnerable Cancer Patient Analysis/README.md]: I have done this too..Please merge it

File: Vulnerable Cancer Patient Analysis/README.md (hunk 2 above)
Comment (CoderOMaster) [reply Vulnerable Cancer Patient Analysis/README.md]: I removed that paragraph since it was obvious in the code that I took 3 epochs

**Review by abhisheks008 — APPROVED** [review]
Approved.
@CoderOMaster

**Comment by CoderOMaster** [comment]
Thank you

**Pull request merged**